docs/3.0rc/automate/events/automations-triggers.mdx

```diff
@@ -749,99 +749,80 @@ This allows you to create automations that respond to these webhook events. From
-    append the data onto 'build_of_names'. Make userID the first column
-    """
-
-@flow
-def create_fake_user_dataset(df):
-  artifact_df = generate_synthetic_user_data(df)
-  print(artifact_df)
-  
-  create_table_artifact(
-      key="fake-user-data",
-      table=artifact_df,
-      description= "Dataset that is comprised of a mix of autogenerated data based on user data"
-  )
 
 if __name__ == "__main__":
-    create_fake_artifact()  
+    upstream_deployment = upstream_flow.to_deployment(name="upstream_deployment")
+    downstream_deployment = downstream_flow.to_deployment(
+        name="downstream_deployment",
+        triggers=[
+            DeploymentEventTrigger(
+                expect={"prefect.flow-run.Completed"},
+                match_related={"prefect.resource.name": "upstream_deployment"},
+            )
+        ],
+    )
+
+    serve(upstream_deployment, downstream_deployment)
 ```
 
-Kick off a deployment with a trigger defined in a `prefect.yaml` file. Specify what to trigger when the event stays in a 
-running state for longer than 30 seconds. 
+First, start the `serve` process to listen for scheduled runs of your deployments:
 
-```yaml
-# Welcome to your prefect.yaml file! You can use this file for storing and managing
-# configuration for deploying your flows. We recommend committing this file to source
-# control along with your flow code.
+```bash
+python event_driven_deployments.py
+```
 
-# Generic metadata about this project
-name: automations-guide
-prefect-version: 3.0.0
+Now, run the upstream deployment and see the downstream deployment kick off after it completes:
 
-# build section allows you to manage and build docker images
-build: null
+```bash
+prefect deployment run upstream-flow/upstream_deployment
+```
 
-# push section allows you to manage if and how this project is uploaded to remote locations
-push: null
+<Tip>
+**Check the event feed**
 
-# pull section allows you to provide instructions for cloning this project in remote locations
-pull:
-- prefect.deployments.steps.set_working_directory:
-    directory: /Users/src/prefect/Playground/marvin-extension
+You can inspect raw events in the event feed in the UI to see what related resources are available to match against.
 
-# the deployments section allows you to provide configuration for deploying flows
-deployments:
-- name: create-fake-user-dataset
-  triggers:
-    - enabled: true
-      match:
-        prefect.resource.id: "prefect.flow-run.*"
-      after: ["prefect.flow-run.Running"]
-      expect: []
-      for_each: ["prefect.resource.id"]
-      parameters:
-        param_1: 10
-      posture: "Proactive"
-  version: null
-  tags: []
-  description: null
-  entrypoint: marvin-extension.py:create_fake_user_dataset
-  parameters: {}
-  work_pool:
-    name: tutorial-process-pool
-    work_queue_name: null
-    job_variables: {}
-  schedule: null
```

Comment:
this whole yaml inclusion I think was confusing because the example never explained the project structure

plus the all python example seems easier to engage with